lib/rest2.js

```diff
@@ -1210,6 +1210,14 @@ class RESTv2 {
     return this._makeAuthRequest('/auth/r/settings', { keys }, cb)
   }
 
+  /**
+   * @param {string[]} keys - keys
+   * @returns {Promise} p
+   */
+  getCoreSettings (keys) {
+    return this._makeAuthRequest('/auth/r/settings/core', { keys })
```

Comment:
Also please add tests for both the model and rest libs, see a sample PR https://github.com/bitfinexcom/bfx-api-node-rest/pull/82

Reply:
I think we should add the same transformer also there, the idea of transformers are to provide more user friendly responses to devs. P.S. we return array responses to decrease network latency since it reduces payload size

Reply:
but adding transformer there should be separate PR